src/System.Security.Permissions/src/System/Diagnostics/EventLogPermissionEntryCollection.cs

```diff
@@ -0,0 +1,26 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections;
+namespace System.Diagnostics
+{
+    [Serializable]
+    public class EventLogPermissionEntryCollection : CollectionBase
+    {
+        internal EventLogPermissionEntryCollection() { }
```

Comment:
@danmosemsft , we've had some ongoing discussions on whether lots of various types should serialize, so I don't have a clear answer for you at this point (though my personal preference is that these types not be serializable). If we decide to change the set of serializable types, I'm sure we'll scrub through corefx to add/subtract serialization as appropriate.